### ISSUE TITLE
No permission to push code to my own project, that I just gas auth -f'd on to ?

### ISSUE THREAD

**Issue by MikeWarren2014** [MaartenDesnouck/google-apps-script#118]
Apparently, I can `gas pull` all I want, but when working on the code, I can't `gas push`. I just tried to `gas auth -f` to force authentication, which it did, but I still "lack permission" upon trying to push again. 

Console output: 

```
CashAmerica@MikeW MINGW64 ~/Desktop/OpenSourceRoads/SendContactEmail
$ gas auth -f
A webbrowser should have opened, to allow 'gas' to:
    'View and manage the files in your Google Drive'
    'Modify your Google Apps Script scripts' behavior'
    'Create and update Google Apps Script projects'
    'Create and update Google Apps Script deployments'

These permissions are necessary for pulling and pushing code from/to your remote scripts.
You are successfully authenticated as '[owner of the script]' [v]

CashAmerica@MikeW MINGW64 ~/Desktop/OpenSourceRoads/SendContactEmail
$ gas push
Pushing 'SendContactEmail' to remote ... [x]
You are not allowed to perform this action.


```

**Comment by MaartenDesnouck** [comment]
@MikeWarren2014, yes, sorry forgot to add the following to the readme and output, will fix that today. 

"User has not enabled the Apps Script API. Enable it by visiting https://script.google.com/home/usersettings then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry."

**Comment by MaartenDesnouck** [comment]
Fixed in 4.1.1
